test/multipoint_routes.cc

```diff
@@ -348,6 +348,36 @@ TEST(MultiPointRoutesBreakThrough, test_mid_break_through_arrive_by) {
   test_mid_break_through(R"(,"date_time":{"type":2,"value":"2016-07-03T08:06"}})");
 }
 
+TEST(MultiPointRoutesBreakThrough, test_mid_break_through_elapsed) {
```

Comment:
we happened to find this on a multipoint route but actually the bug had nothing to do with multipoint routes